state_machine.py

```diff
@@ -441,7 +445,7 @@ def _enter(self, machine):
     def _exit(self, machine):
         pass
 
-    def _update(self, machine, movement):
+    def _update(self, machine):
         machine.intervalForInactivityEventS = FIRST_INTERVAL_INACTIVITY_S
         machine.system.sensor.poll_sensors()
```

Comment:
A function in the system class would be a cleaner way of doing this

Reply:
I will make a system function, which calls the inner sensor function